dataProcessing.py

```diff
@@ -70,32 +70,72 @@ def fill_data(nhits, hitList, db):
 
 
 yeast_name_map = {}
+syst_name_map = {}
+std_name_map = {}
+# Fill name mapping dictionaries
+def fill_name_maps():
+    global yeast_name_map, syst_name_map, std_name_map
+    with open("yeast_names_ref.txt", "r") as f:
+        for line in f:
+            words = line.split()
+            yeast_name = words[0]
+            syst_name = words[1]
+            std_name = ''
```

Comment:
I'd move this line and the next 2 into an `else` clause on the `if`, since they're entirely replaced.